src/utils/sources-tree/treeOrder.js

```diff
@@ -36,6 +37,10 @@ function isExactDomainMatch(part: string, debuggeeHost: string): boolean {
     : part === debuggeeHost;
 }
 
+function isBundler(part: string): boolean {
+  return part === "ng://" || part === "webpack://";
```

Comment:
Can we move `isBundler` to `src/utils/source.js` since it's more of a utility function for sources and less of a treeOrder function?

Reply:
Yeah we can !